src/elements/fields/ButtonGroupField.jsx

```diff
@@ -23,19 +23,23 @@ function ButtonGroupField({
       >
         {servar.metadata.options.map((opt, index) => {
           const imageUrl = servar.metadata.option_images[index];
+          const fieldStyle = applyStyles.getTarget('field');
```

Comment:
since we allow the user to apply different styles at different breakpoints from the dashboard, applyStyles is middleware we use for translating styles stored in the json dict to css styles, handling inheritance, breakpoints, and mobile compatibility. The button group styles are set at https://github.com/feathery-org/feathery-react/blob/master/src/elements/fields/index.jsx#L104, which is where minWidth and maxWidth should be set too (otherwise won't account for breakpoints)

Happy to give more context here if helpful!